fe/fe-core/src/main/java/com/starrocks/scheduler/persist/DropTasksLog.java

```diff
@@ -0,0 +1,37 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.scheduler.persist;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.List;
+
+public class DropTasksLog implements Writable {
+
+    @SerializedName("taskIdList")
+    List<Long> taskIdList;
```

Comment:
what's the difference between List<Long> and List<long>?

Reply:
You mean List<Long> and Long[]?
List<Long> is a dynamic array. It is better to write code to maintain it. The content after serialization and deserialization should be the same.